astropy/io/fits/tests/test_core.py

```diff
@@ -683,7 +683,7 @@ def test_read_open_astropy_gzip_file(self):
         compatibility copy of the ``gzip`` module.
         """
 
-        from ....utils.compat import gzip
+        import gzip
```

Comment:
There's no reason this needs to be an in-line import (there wasn't any reason I can think of previously too, but might as well remove it while we're here).